Sources/SpeziOnboarding/OnboardingFlow/OnboardingViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SwiftUI
+
+public class OnboardingViewController: ObservableObject {
```

Comment:
I would suggest a different name then `OnboardingViewController` here to better fit in the SwiftUI paradigm, I would suggest something like `OnboardingStack` or something more description of the fact that it manages the navigation path. Maybe even `OnboardingNavigationPath`?